tests/integration/framework/util/rand.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"crypto/rand"
+)
+
+// RandomString generates a random string of length n.
+func RandomString(n int) string {
+	const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	bytes := make([]byte, n)
+	rand.Read(bytes)
+	for i, b := range bytes {
+		bytes[i] = letters[b%byte(len(letters))]
```

Comment:
```suggestion
		bytes[i] = letters[rand.Intn(len(letters))]
```

Reply:
`b` contains random bytes so should be ok?

Now, we could argue that using a `char` and doing the modulus [doesn't offer perfect uniformly-distributed random numbers](https://stackoverflow.com/questions/10984974/why-do-people-say-there-is-modulo-bias-when-using-a-random-number-generator), but this is just a test :)

Reply:
I think it should be fine, but have changed it to use `rand.Int` anyway.